tests/rpc_test.go

```diff
@@ -94,26 +94,22 @@ func call(t *testing.T, method string, params interface{}) (*Response, error) {
 	/* #nosec */
 	res, err := http.Post(ETHERMINT_NODE_HOST, "application/json", bytes.NewBuffer(req))
 	if err != nil {
```

Comment:
these should all be `require.NoError(t, err)` instead of the error checks

Reply:
done